### PR TITLE
fix: translate only translatable fields

### PR DESCRIPTION
**Pull request by e-lobo**

### CODE REVIEW

File: frappe_graphql/utils/resolver/document_resolver.py [hunk 1]
```diff
@@ -67,8 +67,10 @@ def _get_value(fieldname, ignore_translation=False):
 
         # ignore_doc_resolver_translation might be helpful for overriding document_resolver
         # which might be a simple wrapper around this function (document_resolver)
+        field_name_exits = meta.get_field(info.field_name)
```
Comment (fahimalizain) [inline frappe_graphql/utils/resolver/document_resolver.py]:
```suggestion
        df = meta.get_field(info.field_name)
```

**Review by Abadulrehman — APPROVED** [review]

File: frappe_graphql/utils/resolver/document_resolver.py (hunk 1 above)
Comment (e-lobo) [reply frappe_graphql/utils/resolver/document_resolver.py]: @fahimalizain cannot do that df is declared and assigned in the outer scope 🥶

Comment (fahimalizain) [reply frappe_graphql/utils/resolver/document_resolver.py]: @e-lobo why not re-use the same variable ? it is accessible right ?

Comment (e-lobo) [reply frappe_graphql/utils/resolver/document_resolver.py]: @fahimalizain im not sure i can do that. check the usages of `_get_value(fieldname)` the `fieldname` is unpredictable. please clarify ?

Comment (fahimalizain) [reply frappe_graphql/utils/resolver/document_resolver.py]: @e-lobo yes, nice catch 👍🏼 
I would say you could rename it to `_df` or something at least.  `field_name_exits ` can only ever be a `boolean` 😄

Comment (e-lobo) [reply frappe_graphql/utils/resolver/document_resolver.py]: @fahimalizain updated ✅

**Review by fahimalizain — APPROVED** [review]
Looks good 👍🏼

**Pull request merged**